src/app/shared/ui/spinner-dots.component.scss

```diff
@@ -0,0 +1,43 @@
+@import './src/styles';
```

Comment:
nie powinniśmy importować globalnych styli do komponentu, czemu to robimy?

Reply:
Ja u siebie na tej gałęzi nie w idze tego importu, o ile pamiętam usunąłem go w pierwszym sprincie gdy tworzyłem ten komponent. Na developie też nie ma tego importu.